.github/workflows/test.yml

```diff
@@ -208,14 +208,10 @@ jobs:
     needs: [build-test-bins, build-test-artifacts-postgres]
     strategy:
       matrix:
-        #version: [02,99]
-        version: [02]
         include:
-          - version: 02
-            test-name: test_native_demo_basic
-
-          # - version: 99
-          #   test-name: test_native_demo_upgrade
+          - test-name: test_native_demo_base
+          - test-name: test_native_demo_pos_base
+          - test-name: test_native_demo_pos_upgrade
```

Comment:
can we also add these options to justfile for easy local testing?

Reply:
There are no extract config / env vars required now so `cargo test test_native_demo_base` already works. Is that sufficient?